sample/src_polymer/elements/enhanced-photography-pages/enhanced-photo-page.html

```diff
@@ -0,0 +1,20 @@
+<link rel="import" href="../elements.html">
+<dom-module id="enhanced-photo-page">
+  <template>
+    <style include="shared-style">
```

Comment:
@jondong  Should change `shared-style`  ---> `shared-styles`, for all 5 .html files in `elements/enhanced-photography-pages`